style.css

```diff
@@ -0,0 +1,1226 @@
+  font-size: 16px;
+}
+
+.article-details ul {
+  display: flex;
+}
+
+.article-details ul li {
+  margin-left: 0;
+  margin-right: 5px;
+}
+
+.article-details ul li > a,
+.article-details ul li > span {
+  font-size: 12px;
+  color: #aaa;
+}
+
+.funding a:nth-child(3) {
+  font-size: 12px;
+  font-weight: 400;
+  padding: 16px;
+}
+
+.funding-rounds {
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+}
+
+.funding-rounds-header {
+  color: #f42;
+  margin-bottom: 24px;
+}
+
+.funding-rounds-header h2 {
+  font-style: sans-serif;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+}
+
+.funding-rounds article {
+  min-height: 60px;
+  background: #fffaf8;
+  margin-bottom: 24px;
+}
+
+.funding-4 {
+  display: none;
+}
+
+.latest-deals header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.deal-1 {
+  background: url("images/latest-deals-1.jpg");
+}
+
+.deal-2 {
+  background: url("images/latest-deals-2.jpg");
+}
+
+.deal-3 {
+  background: url("images/latest-deals-3.jpg");
+}
+
+.deal-4 {
+  background: url("images/latest-deals-4.jpg");
+}
+
+.deal-5 {
+  background: url("images/latest-deals-5.jpg");
+}
+
+.deal-6 {
+  background: url("images/latest-deals-6.jpg");
+}
+
+.deal-7 {
+  background: url("images/latest-deals-7.jpg");
+}
+
+.deal-8 {
+  background: url("images/latest-deals-8.jpg");
+}
+
+del {
+  font-size: 12px;
+  color: #aaa;
+}
+
+footer {
+  border-top: 1px solid black;
+}
+
+.footer-links {
+  background: #131313;
+}
+
+.social-links {
+  margin: 0 auto;
+  padding-top: 3rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.site-links {
+  padding: 1.8rem 0 1.8rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.tar {
+  background: #333;
+  height: 45px;
+  width: 45px;
+  font-size: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  border: 2px solid #333;
+  border-radius: 50%;
+}
+
+.tar:hover {
+  background: #131313;
+}
+
+.legal-copyright {
+  background: #000;
+  color: #aaa;
+  padding: 3rem 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+@media screen and (max-width: 767px) {
+  .nav-bar-holder {
+    height: 3rem;
+  }
+
+  .secondary-menu {
+    display: none;
+  }
+
+  #main-menu-links {
+    display: none;
+  }
+
+  nav {
+    width: 100%;
+    position: fixed;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 3rem;
+    background: #fff;
+    z-index: 999;
+  }
+
+  .tnw-logo {
+    height: 3rem;
+  }
+
+  .tnw-logo img {
+    height: 1rem;
+    width: 66px;
+  }
+
+  .search-menu {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+  }
+
+  .search-menu li {
+    height: 100%;
+    width: 3rem;
+    border-left: 1px solid #eee;
+  }
+
+  .search {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
- [ ] Make the adjustments to avoid the repetition of properties.

Reply:
Done thanks